addons/spreadsheet/static/src/helpers/model.js

```diff
@@ -69,6 +70,15 @@ export async function freezeOdooData(model) {
             const sheetId = sheet.id;
             const position = { sheetId, col, row };
             const evaluatedCell = model.getters.getEvaluatedCell(position);
+            if (
+                evaluatedCell.type === "text" &&
+                (isNumber(evaluatedCell.value, DEFAULT_LOCALE) ||
+                    isDateTime(evaluatedCell.value, DEFAULT_LOCALE))
+            ) {
+                cell.content = `="${evaluatedCell.value}"`;
+            } else {
+                cell.content = evaluatedCell.value.toString();
+            }
```

Comment:
this `else` kills every formula :/ We should preserve formulas if they do not contain dynamic data, right ?

Reply:
yup, I wonder why I put this there, I think it was meant to be inside the following if statement